delfin/task_manager/scheduler/schedulers/telemetry/telemetry_job.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+
+import six
+from oslo_log import log
+from oslo_utils import importutils
+from oslo_utils import uuidutils
+
+from delfin import db
+from delfin.task_manager.scheduler import scheduler
+
+LOG = log.getLogger(__name__)
+
+
+class TelemetryJob(object):
+    def __init__(self, ctxt):
+        # create the object of periodic scheduler
+        self.schedule = scheduler.Scheduler.get_instance()
+        # Reset last run time of tasks to restart scheduling
+        task_list = db.task_get_all(ctxt)
+        for task in task_list:
+            db.task_update(ctxt, task['id'], {'last_run_time': None})
+
+    def __call__(self, ctx):
+        """ Schedule the collection tasks based on interval """
+        try:
+
+            filters = {'last_run_time': None}
```

Comment:
Need to check whether task table has any update in already scheduled jobs, like deleted=true , https://github.com/sodafoundation/delfin/pull/513 will mark deleted=true ,  may be we can keep a map of DB task_id to scheduler job_id .. and then remove the job for deleted task entries